Centaurus.Domain/Extensions/IExtension.cs

```diff
@@ -8,5 +8,7 @@ namespace Centaurus.Domain
     public interface IExtension
     {
         Task Init(Dictionary<string, string> settings);
+
+        Task Terminate();
```

Comment:
There is a standard `IDisposable` interface for this purpose. Should be `IExtension: IDisposable`